app/lib/actions/pulp3/repository/presenters/content_unit_presenter.rb

```diff
@@ -12,7 +12,16 @@ def progress
           def humanized_details
             ret = []
             ret << _("Cancelled.") if cancelled?
-            ret << _("New Content Units: %s") % finished_units
+            ret << _("Content Units found: %s") % finished_units
```

Comment:
This line **"Content Units found:"** results in inconsistent data. We have different messages for all content types which can return this value. For example: In Ansible the message would be "Downloading Collections", in Files it would be "Parsing Metadata". 
We could possibly leave this out to make this generic enough or figure out a way to track different messages for different content type which might take some additional work. Looking for ideas on this one.